tests/test_integrator/test_fiat_adapter.py

```diff
@@ -172,7 +172,24 @@ def test_fiat_adapter_measures(cleanup_database):
     )
 
 
-def test_fiat_return_periods(cleanup_database):
+def test_fiat_raise_datum():
+    test_toml = (
+        test_database
+        / "charleston"
+        / "input"
+        / "scenarios"
+        / "current_extreme12ft_raise_datum"
+        / "current_extreme12ft_raise_datum.toml"
+    )
+
+    assert test_toml.is_file()
+
+    # use event template to get the associated Event child class
+    test_scenario = Scenario.load_file(test_toml)
+    test_scenario.run()
+
```

Comment:
ideally you would do just a unit test and test this one function. E.g. that certain buildings in the exposure csv are higher now after applying the measure. 
Because the model is run it takes really long and we are not checking whether the measure was actually applied.